cdk/v2/destinations/algolia/rtWorkflow.yaml

```diff
@@ -0,0 +1,65 @@
+bindings:
+- path: ../../../../v0/destinations/algolia/config
+
+steps:
+- name: validateInput
+  template: |
+    (
+      $assert($type($) = "array" and $count($) > 0, "Invalid event array")
+    )
+
+- name: prepareContext
+  template:
+    $setContext("batchMode", true)
+
+- name: transform
+  externalWorkflow:
+    path: ./procWorkflow.yaml
+  loopOverInput: true
+
+- name: processTransformOutput
```

Comment:
Can you check latest Pinterest procWorkflow.yaml, simplified these for better readability.

May be we can reuse the template using templatePath feature

Reply:
For now, I've implemented it independently. Let's refactor it later.